tests/delete/tests.py

```diff
@@ -356,6 +356,35 @@ def test_delete_with_keeping_parents(self):
         self.assertFalse(RChild.objects.filter(id=child.id).exists())
         self.assertTrue(R.objects.filter(id=parent_id).exists())
 
+    def test_delete_qs_returns_num_rows(self):
```

Comment:
Are the different models testing two different code paths? It's unclear to me why the second test isn't sufficient for testing the number of deleted rows too.